src/frontend/Player/createPlyrPlayer.ts

```diff
@@ -59,6 +59,13 @@ export const createPlyrPlayer = (
       }
     }
     xapiStatement.initialized(contextExtensions);
+
+    const playLarge: Nullable<HTMLButtonElement> = document.querySelector(
+      '.plyr .plyr__control--overlaid',
+    );
+    if (!!playLarge) {
+      playLarge.tabIndex = -1;
```

Comment:
Same. Should we have a test or at least comment to secure it?

Reply:
I didn't find how to test it. I tried to render the player but unfortunately it didn't work and the plyr repo does not have tests I can learn to understand how to do it... 
The comment is added.